### PR TITLE
Passing the label context to regional module

### PR DESCRIPTION
**Pull request by lazzurs**
In the last change I did not add passing the
label context to the regional module which
causes the IAM role to have a name of
-iam_for_lambda rather than adding the further
context to make sure it is unique per
environment.

### CODE REVIEW

**Comment by lazzurs** [comment]
Once this is merged it would be useful if there was another release of the module so I could move back to using the Terraform registry.

Happy to help out maintaining this Terraform module.

**Review by cristim — APPROVED** [review]
Thanks for your contribution and for volunteering to help maintain the Terraform module. Any help would be very welcome.

I'll try try to make a new release later today and I'll grant you maintainer permissions to the repository.

**Comment by cristim** [comment]
Released 0.1.6 with the latest code changes

**Pull request merged**
